src/styles/reset.css

```diff
@@ -0,0 +1,10 @@
+* {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
```

Comment:
Setting the font-family for all elements may not be the best approach, consider applying it to the body element only.